modules/oas-page-builder/tests/unit/services/pageBuilder.test.ts

```diff
@@ -21,7 +21,10 @@ jest.mock('../../../src/services/database', () => ({
 }));
 
 // Helper function for concatendated output path
-const getExpectedOutputPath = (destination: string, pageSlug: string) => `${destination}/${pageSlug}/index.html`;
+const getExpectedOutputPath = (destination: string, pageSlug: string, apiVersion?: string, resourceVersion?: string) =>
+  `${destination}/${pageSlug}${apiVersion ? `${apiVersion ? `/v${apiVersion.split('.')[0]}` : ''}` : ''}${
+    resourceVersion ? `/${resourceVersion}` : ''
```

Comment:
This shouldn't be an issue, but for consistency maybe add the extra truthy check here as well!